test/Maps/MapReader.test.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Maps/MapReader.h"
+#include "Maps/MapType.h"
+#include "gtest/gtest.h"
```

Comment:
We should probably use angle brackets for `<gtest/gtest.h>`. It's not part of our project, nor is it part of our source tree.

Reply:
Good call here.